contracts/src/v0.8/automation/dev/v2_3/AutomationRegistryLogicA2_3.sol

```diff
@@ -201,6 +207,7 @@ contract AutomationRegistryLogicA2_3 is AutomationRegistryBase2_3, Chainable {
         billingToken.safeTransfer(destination, balanceToTransfer);
```

Comment:
could we move this transfer to outside the loop and drop the `if` clause? I think that would look cleaner.

Reply:
yes, agreed.